### ISSUE TITLE
Shorten the announcements API route

### ISSUE THREAD

**Issue by kurtaking** [procore-oss/backstage-plugin-announcements#23]
The existing API routes are duplicative -  `.../announcements/announcements/`

For example, the URL for local development resolves to
```
http://localhost:7007/api/announcements/announcements
```

It would be nice to shorten this to `http://localhost:7007/api/announcements/`, and I believe all it will take is shortening the root route ref from `/announcement` to `/`.

**Comment by Nereis** [comment]
I don't think it's a good idea to shortcut the root as it will collide with other plugins and backstage won't be able to know to which plugin it has to root the request.

**Comment by kurtaking** [comment]
@Nereis, I think we only need to do this for the GET request, actually. You are correct in that I do not want to shorten the root route ref. 👍🏼
